locale/en/knowledge/intermediate/how-to-log.md

```diff
@@ -57,17 +57,19 @@ Here is an example of setting up a `winston` logger.  This example includes most
 
       require('winston-riak').Riak;
       require('winston-mongo').Mongo;
+      require('winston-couchdb').Couchdb;
```

Comment:
As of winston@0.6.0 the CouchDB transport has been broken out into a new module: winston-couchdb. So winston dont have `winston.transports.Couchdb` default